src/sot-dynamic.cpp

```diff
@@ -308,9 +328,12 @@ dg::SignalTimeDependent<ml::Vector,int>& Dynamic::accelerationsSOUT( const std::
 
```

Comment:
time has to be given for signal access instead of "1"
getPinochiosPos/Val/Acc must take "time"